clients/include/hipblas_common.yaml

```diff
@@ -594,6 +595,7 @@ Defaults:
   backend_flags: ALL_BACKEND
   api: C
   initialization: rand_int
+  bad_arg_all: true
   compute_type_gemm: 2
```

Comment:
Okay so bad_arg_all is just to let you test the expected failures for inconsistent backend?   Guess it make sense, but a || backend_all could be another option.   This is just likely applicable to bad_arg I assume

Reply:
Yep, that's right. I'll leave it for now and see how the rest of L2/L3 bad args looks.